acceptance/tests/options/custom_facts.rb

```diff
@@ -1,4 +1,5 @@
 test_name "custom fact commandline options (--no-custom-facts and --custom-dir)"
+confine :except, :platform => /^cisco-/
```

Comment:
Can you add a note about why this is disabled? I assume it's because we don't have Ruby for Cisco environments.